{
 "about": {
  "channels": [
   "https://conda.anaconda.org/conda-forge/osx-64",
   "https://conda.anaconda.org/conda-forge/noarch",
   "https://repo.continuum.io/pkgs/free/osx-64",
   "https://repo.continuum.io/pkgs/free/noarch",
   "https://repo.continuum.io/pkgs/pro/osx-64",
   "https://repo.continuum.io/pkgs/pro/noarch"
  ],
  "conda_build_version": "2.1.10",
  "conda_env_version": "4.2.13",
  "conda_private": false,
  "conda_version": "4.2.13",
  "dev_url": "https://github.com/pytest-dev/pytest-asyncio",
  "env_vars": {
   "CIO_TEST": "<not set>",
   "CONDA_DEFAULT_ENV": "root",
   "CONDA_ENVS_PATH": "<not set>",
   "DYLD_LIBRARY_PATH": "<not set>",
   "PATH": "/Users/travis/miniconda3/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p643/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p643@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p643/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin",
   "PYTHONHOME": "<not set>",
   "PYTHONPATH": "<not set>"
  },
  "home": "http://github.com/pytest-dev/pytest-asyncio",
  "license": "Apache 2.0",
  "offline": false,
  "root_pkgs": [
   "anaconda-client-1.6.3-py36_0",
   "conda-forge::beautifulsoup4-4.5.3-py36_0",
   "cffi-1.9.1-py36_0",
   "conda-forge::chardet-3.0.2-py36_1",
   "conda-forge::clyent-1.2.1-py36_0",
   "conda-forge::conda-4.2.13-py36_0",
   "conda-forge::conda-build-2.1.10-py36_0",
   "conda-forge::conda-env-2.6.0-0",
   "conda-forge::conda-forge-build-setup-4.4.4-0",
   "conda-forge::conda-verify-2.0.0-py36_0",
   "cryptography-1.7.1-py36_0",
   "conda-forge::filelock-2.0.6-py36_0",
   "idna-2.2-py36_0",
   "conda-forge::jinja2-2.9.5-py36_0",
   "conda-forge::markupsafe-0.23-py36_1",
   "openssl-1.0.2k-0",
   "pip-9.0.1-py36_1",
   "conda-forge::pkginfo-1.2.1-py36_0",
   "pyasn1-0.1.9-py36_0",
   "pycosat-0.6.1-py36_1",
   "pycparser-2.17-py36_0",
   "pycrypto-2.6.1-py36_4",
   "pyopenssl-16.2.0-py36_0",
   "python-3.6.0-0",
   "conda-forge::python-dateutil-2.6.0-py36_0",
   "conda-forge::pytz-2017.2-py36_0",
   "conda-forge::pyyaml-3.12-py36_1",
   "readline-6.2-2",
   "requests-2.12.4-py36_0",
   "ruamel_yaml-0.11.14-py36_1",
   "setuptools-27.2.0-py36_0",
   "six-1.10.0-py36_0",
   "sqlite-3.13.0-0",
   "tk-8.5.18-0",
   "wheel-0.29.0-py36_0",
   "xz-5.2.2-1",
   "yaml-0.1.6-0",
   "zlib-1.2.8-3"
  ],
  "summary": "Pytest support for asyncio"
 },
 "conda_build_config": {},
 "files": [
  "lib/python2.7/site-packages/pytest_asyncio/plugin.py",
  "lib/python2.7/site-packages/pytest_asyncio-0.5.0-py2.7.egg-info/PKG-INFO",
  "lib/python2.7/site-packages/pytest_asyncio/__init__.py"
 ],
 "index": {
  "arch": "x86_64",
  "build": "py27_0",
  "build_number": 0,
  "depends": [
   "pytest >=3.0.2",
   "python 2.7*"
  ],
  "license": "Apache 2.0",
  "name": "pytest-asyncio",
  "platform": "osx",
  "subdir": "osx-64",
  "version": "0.5.0"
 },
 "metadata_version": 1,
 "name": "pytest-asyncio",
 "raw_recipe": "{% set name = \"pytest-asyncio\" %}\n{% set version = \"0.5.0\" %}\n{% set sha256 = \"ce457148d9a8337679b19742d37f77c5e4eb53a48b626e3ce19464fa1cc25d0f\" %}\n\npackage:\n  name: {{ name|lower }}\n  version: {{ version }}\n\nsource:\n  fn: {{ name }}-{{ version }}.tar.gz\n  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz\n  sha256: {{ sha256 }}\n\nbuild:\n  number: 0\n  script: python setup.py install --single-version-externally-managed --record record.txt\n\nrequirements:\n  build:\n    - python\n    - setuptools\n  run:\n    - python\n    - pytest >=3.0.2\n\ntest:\n    imports:\n      - pytest_asyncio\n\nabout:\n  home: http://github.com/pytest-dev/pytest-asyncio\n  license: Apache 2.0\n  #license_file: LICENSE # License file is missing in package\n  summary: 'Pytest support for asyncio'\n\n  dev_url: https://github.com/pytest-dev/pytest-asyncio\n\nextra:\n  recipe-maintainers:\n    - sodre\n",
 "rendered_recipe": {
  "about": {
   "dev_url": "https://github.com/pytest-dev/pytest-asyncio",
   "home": "http://github.com/pytest-dev/pytest-asyncio",
   "license": "Apache 2.0",
   "summary": "Pytest support for asyncio"
  },
  "build": {
   "noarch": "",
   "noarch_python": false,
   "number": "0",
   "script": "python setup.py install --single-version-externally-managed --record record.txt",
   "string": "py27_0"
  },
  "extra": {
   "final": true,
   "recipe-maintainers": [
    "sodre"
   ]
  },
  "package": {
   "name": "pytest-asyncio",
   "version": "0.5.0"
  },
  "requirements": {
   "build": [
    "ca-certificates 2017.1.23 1",
    "certifi 2017.4.17 py27_0",
    "ncurses 5.9 10",
    "openssl 1.0.2k 0",
    "python 2.7.13 1",
    "readline 6.2 0",
    "setuptools 33.1.1 py27_0",
    "sqlite 3.13.0 1",
    "tk 8.5.19 1",
    "zlib 1.2.11 0"
   ],
   "run": [
    "python",
    "pytest >=3.0.2"
   ]
  },
  "source": {
   "fn": "pytest-asyncio-0.5.0.tar.gz",
   "sha256": "ce457148d9a8337679b19742d37f77c5e4eb53a48b626e3ce19464fa1cc25d0f",
   "url": "https://pypi.io/packages/source/p/pytest-asyncio/pytest-asyncio-0.5.0.tar.gz"
  },
  "test": {
   "imports": [
    "pytest_asyncio"
   ]
  }
 },
 "version": "0.5.0"
}